src/css/style.css

```diff
@@ -51,6 +51,15 @@
   margin: 25px;
 }
 
+.card:hover {
+  margin-top: 0px;
+  margin-bottom: 50px;
+  box-shadow: 0 28px 18px 0 rgba(0, 0, 0, 0.2);
+
+  -webkit-transition: 150ms ease-in-out;
+  transition: 150ms ease-in-out;
```

Comment:
Move this in the `.card` selector so that it also animating when on mouse leave. Also, can you add the time (250ms or 350ms) for slower animation?